app/assets/stylesheets/components/_layout.scss

```diff
@@ -75,3 +75,32 @@
     }
   }
 }
+
+.back-to-top {
+  display: none;
+  float: right;
+  margin-top: 1.5rem;
+  position: sticky;
+  position: -webkit-sticky;
```

Comment:
Minor change here, I'd reorder these two lines:

```scss
position: -webkit-sticky;
position: sticky;
```

The difference there is that browsers that support the spec (`position: sticky`) will use that value instead of the WebKit-specific vendor prefixed value.